spec/models/course_spec.rb

```diff
@@ -5,9 +5,17 @@
   let!(:instance) { create(:instance) }
   with_tenant(:instance) do
     it { is_expected.to belong_to(:creator) }
-    it { is_expected.to have_many(:course_users).inverse_of(:course).dependent(:destroy) }
-    it { is_expected.to have_many(:users).through(:course_users) }
-    it { is_expected.to have_many(:announcements).inverse_of(:course).dependent(:destroy) }
+    it { is_expected.to have_many(:course_users)
+                        .inverse_of(:course)
+                        .dependent(:destroy) }
+    it { is_expected.to have_many(:users)
+                        .through(:course_users) }
```

Comment:
Expression at 12, 49 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 20) spaces for indenting an expression spanning multiple lines.